.github/workflows/branch-name-validator.yml

```diff
@@ -9,5 +9,5 @@ jobs:
       - uses: actions/checkout@v3
       - uses: deepakputhraya/action-branch-name@master
         with:
-          regex: '^(no-ticket|dependabot|IPB-[0-9]+)/' # start branch with either 'no-ticket/' or 'dependabot' or 'IPB-NUMBER/'
-          ignore: main
+          regex: '^(no-ticket|renovate|IPB-[0-9]+)/' # start branch with either 'no-ticket/' or 'IPB-NUMBER/', allowing for bots
```

Comment:
TIL that renovate sometimes opens PRs with `renovate-` prefix, which wouldn't match here